include/sway/tree/container.h

```diff
@@ -76,12 +76,14 @@ struct sway_container {
 	enum sway_container_layout layout;
 	enum sway_container_layout prev_layout;
 
+	bool is_sticky;
+
 	// For C_ROOT, this has no meaning
-	// For C_OUTPUT, this is the output position in layout coordinates
-	// For other types, this is the position in output-local coordinates
+	// For other types, this is the position in layout coordinates
```

Comment:
In fact, even for C_ROOT, this is the position in layout coordinates (for C_ROOT, it's the extents of the whole output layout).